### DISCUSSION TITLE
扣币详细方案

### DISCUSSION THREAD

**Discussion by jzmexec** [filecoin-project/community-china#143] [新手上路]
有问题请教各位大佬
1. 掉算力后扣币量的算法？
2. 退算力质押是否能返回，扣币的算法？

**Comment by Chenhongwe** [comment]
Fil算力填充，240/T，包半年运维，送七牛商业存储软件一年，坐标杭州，欢迎骚扰 phone：15255579473（同V）

**Comment by SeanLau0811** [comment]
需要了解file的任何信息，可以咨询微信：xiaofu0811

**Comment by zhaobo443** [comment]
小型机房建设，专业IPFS节点搭建，分币系统app定制全套filecoin,包括测ceph存储技术，长期合作全部免费奉送，长期维护只抽取5%服务费！联系电话：19965407481
